tests/mpi/renumber_cuthill_mckee.cc

```diff
@@ -22,6 +22,7 @@
 #include "../tests.h"
 #include <deal.II/base/logstream.h>
 #include <deal.II/base/mpi.h>
+#include <deal.II/base/mpi.templates.h>
```

Comment:
What's the reason for this? What instantiation do we need in this test that isn't already provided?